src/modules/LanUdevAuto.rb

```diff
@@ -180,7 +180,7 @@ def Import(settings)
           createUdevFromIfaceName(Ops.get_list(settings, "interfaces", []))
         )
       else
-        @udev_rules = Ops.get_list(settings, "net-udev", [])
+        @udev_rules = settings["net-udev"] || []
```

Comment:
I do not get how it fixes internal error. For me it looks equivalent or I overlook something?